### DISCUSSION TITLE
Teardown issue with failed tests via vitest

### DISCUSSION THREAD

**Discussion by perillai** [webdriverio/webdriverio#8656] [Q&A]
While running "test:unit" and if the test fails, the teardown in globalSetup.js does not happen. So, the 'tmp_got' package is not getting renamed to 'got' and throws an issue when we start the unit test again. Is anybody else facing this issue?

**Comment by christian-bromann** [comment]
> Is anybody else facing this issue?

It never failed on my machine. Even if you stop the test with CMD + c the teardown function is called properly.
